packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -658,15 +644,24 @@ contract Bridge is EssentialContract, IBridge {
     {
         unchecked {
             // https://github.com/ethereum/execution-specs/blob/master/src/ethereum/cancun/vm/gas.py#L128
-            uint256 words = _dataLength / 32 + 1;
-            uint256 memoryGasCost = words * 3 + words * words / 512;
-
-            // Need to add the gas cost of accessing the message.to account. This currently
-            // corresponds to 2600 gas if the account is cold, and 100 otherwise.
 
-            // Also need to add the cost of transferring a nonzero msg.value. This cost is currently
-            // 9000, but provides a 2300 gas stipend to the called contract.
-            result_ = gasleft() * 63 < _minGas * 64 + (memoryGasCost + 9300) * 63;
+            // We can get the actual memory allocated with `msize()`, but that will require
+            // yul optimizer turned off, which will make other contract's code size too large.
```

Comment:
So we're trying to calculate the gas cost to expand the memory to setup the call on the bridge side right? I previously assumed it was the potential memory expansion inside the call, but that actually makes less sense.

Reply:
We are trying to measure the memory cost of *setting up* the call, not the memory cost inside the call.